inspirehep/modules/theme/jinja2filters.py

```diff
@@ -511,12 +512,18 @@ def publication_info(record):
             conference_recid = None
             parent_recid = None
             if 'conference_record' in pub_info:
-                conference_rec = replace_refs(pub_info['conference_record'],
-                                              'es')
+                if isinstance(pub_info['conference_record'], jsonref.JsonRef):
+                    conference_rec = dict(pub_info['conference_record'])
+                else:
+                    conference_rec = replace_refs(pub_info['conference_record'],
+                                                  'es')
                 if conference_rec and conference_rec.get('control_number'):
                     conference_recid = conference_rec['control_number']
             if 'parent_record' in pub_info:
-                parent_rec = replace_refs(pub_info['parent_record'], 'es')
+                if isinstance(pub_info['parent_record'], jsonref.JsonRef):
+                    parent_rec = dict(pub_info['parent_record'])
+                else:
+                    parent_rec = replace_refs(pub_info['parent_record'], 'es')
```

Comment:
This thing souldn't recursively resolve all references unless parent_rec is fully serialized to json. If we only access some fields in the template I think it's safe to leave the code as is.